libraries/apollo-debug-server/src/commonMain/kotlin/com/apollographql/apollo3/debugserver/internal/graphql/GraphQL.kt

```diff
@@ -0,0 +1,191 @@
+package com.apollographql.apollo3.debugserver.internal.graphql
+
+import com.apollographql.apollo3.ApolloClient
+import com.apollographql.apollo3.annotations.ApolloAdapter
+import com.apollographql.apollo3.annotations.ApolloObject
+import com.apollographql.apollo3.annotations.GraphQLName
+import com.apollographql.apollo3.api.Adapter
+import com.apollographql.apollo3.api.CustomScalarAdapters
+import com.apollographql.apollo3.api.ExecutionContext
+import com.apollographql.apollo3.api.json.JsonReader
+import com.apollographql.apollo3.api.json.JsonWriter
+import com.apollographql.apollo3.api.json.writeObject
+import com.apollographql.apollo3.ast.toGQLDocument
+import com.apollographql.apollo3.ast.toSchema
+import com.apollographql.apollo3.cache.normalized.api.CacheKey
+import com.apollographql.apollo3.cache.normalized.api.Record
+import com.apollographql.apollo3.cache.normalized.apolloStore
+import com.apollographql.apollo3.debugserver.internal.graphql.execution.ApolloDebugServerAdapterRegistry
+import com.apollographql.apollo3.debugserver.internal.graphql.execution.ApolloDebugServerResolver
+import com.apollographql.apollo3.execution.ExecutableSchema
+import com.apollographql.apollo3.execution.GraphQLRequest
+import com.apollographql.apollo3.execution.GraphQLRequestError
+import com.apollographql.apollo3.execution.parsePostGraphQLRequest
+import kotlinx.coroutines.runBlocking
+import okio.Buffer
+import kotlin.reflect.KClass
+
+internal expect fun getExecutableSchema(): String
```

Comment:
Nit: rename to `getSchema()` as this one is not executable (it doesn't contain resolvers)